### DISCUSSION TITLE
`URL` field can't be displayed `asHtml`

### DISCUSSION THREAD

**Discussion by kichetof** [laravel/nova-issues#4951] [Q&A] [answered]
- Laravel Version: 9.36.1
- Nova Version: 4.16.1
- PHP Version: 8.1.11
- Database Driver & Version: Sail mysql
- Operating System and Version: macOS Sail
- Browser type and version: Brave

### Description:
I would like to display an `URL` field with custom html, and it doesn't work as expected.

URL field display the name of the field instead of the custom html:

<img width="406" alt="Capture d’écran 2022-10-19 à 09 06 50" src="https://user-images.githubusercontent.com/12150996/196620514-3e135be1-f30d-4273-91a4-84b1d50cc8e4.png">

If you replace `URL` by `Text` field, custom html are correctly displayed.

### Detailed steps to reproduce the issue on a fresh Nova installation:
```php
URL::make('test URL', function () {
    return '<a href="https://laravel.com">laravel.com</a>';
})->asHtml(),

Text::make('test TEXT', function () {
    return '<a href="https://laravel.com">laravel.com</a>';
})->asHtml(),
```

**Comment by crynobone** [comment, marked as answer]
Computed value should only return the URL while you can customise the HTML via `displayUsing()`

```php
URL::make('test URL', fn () => 'https://laravel.com')
                ->displayUsing(fn ($value) => '<a href="'.$value.'">laravel.com</a>')
                ->asHtml(),
```
